components/base/kinematicbase/differentialDrive.go

```diff
@@ -122,6 +128,12 @@ func (ddk *differentialDriveKinematics) GoToInputs(ctx context.Context, desired
 	cancelContext, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	if ddk.Localizer == nil {
```

Comment:
[opt] can you add a comment describing how the behavior of the func is different should there be no localizer?

Reply:
Done